migrator/migrations/n_57_to_n_58_postgres_remove_clustercve_permission/migration.go

```diff
@@ -0,0 +1,62 @@
+package n57ton58
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	permissionsetpostgresstore "github.com/stackrox/rox/migrator/migrations/n_57_to_n_58_postgres_remove_clustercve_permission/permissionsetpostgresstore"
+	"github.com/stackrox/rox/migrator/types"
+)
+
+const (
+	batchSize = 500
+)
+
+var (
+	migration = types.Migration{
+		StartingSeqNum: 168,
+		VersionAfter:   &storage.Version{SeqNum: 169},
+		Run: func(databases *types.Databases) error {
+			err := cleanupPermissionSets(databases.PostgresDB)
+			if err != nil {
+				return errors.Wrap(err, "updating PermissionSet schema")
+			}
+			return nil
+		},
+	}
+	clusterCVEResourceName = "ClusterCVE"
+)
+
+func cleanupPermissionSets(db *pgxpool.Pool) error {
+	ctx := context.Background()
+	permissionSetStore := permissionsetpostgresstore.New(db)
+	permissionSetsToInsert := make([]*storage.PermissionSet, 0, batchSize)
+	err := permissionSetStore.Walk(ctx, func(obj *storage.PermissionSet) error {
+		if _, ok := obj.GetResourceToAccess()[clusterCVEResourceName]; ok {
+			delete(obj.ResourceToAccess, clusterCVEResourceName)
+			permissionSetsToInsert = append(permissionSetsToInsert, obj)
```

Comment:
The changelog states `...the permission 'Cluster' replaces the deprecated permission 'ClusterCVE'`. However, you are only deleting the `ClusterCVE` resource. There is an assumption that the permission set contains resource `Cluster`.

I think either you need to changelog the changelog or make sure Cluster resource exists in the permission set.